docs/reference/commandline/volume_create.md

```diff
@@ -54,17 +54,25 @@ different volume drivers may do different things (or nothing at all).
 
 The built-in `local` driver on Windows does not support any options.
 
-The built-in `local` driver on Linux accepts options similar to the linux `mount`
-command:
+The built-in `local` driver on Linux accepts options similar to the linux `mount` command. You can provide multiple options using a comma-separated list.
```

Comment:
@thaJeztah only first one is valid. Thoughts on correct way of explaining that? This is relevant for all parameters because they're treated as keys which need to be unique.